src/Time/Series.hs

```diff
@@ -174,3 +181,79 @@ instance ( KnownRatName unit
 -}
 unitsF :: forall unit . KnownRatName unit => Time unit -> String
 unitsF = seriesF @AllTimes
+
+{- | Class for time parsing.
+
+__Examples__
+
+>>> seriesP @'[Day, Hour, Minute, Second] @Minute "2d18h40m"
+Just (4000m)
+
+>>> seriesP @'[Day, Minute, Second] @Minute "2d1120m"
+Just (4000m)
+
+>>> seriesP @'[Hour, Minute, Second] @Second "1h1s"
+Just (3601s)
+
+>>> seriesP @'[Hour, Second, Millisecond] @Minute "90s"
+Just (1+1/2m)
+
+>>> seriesP @'[Hour, Second] @Second "11ns"
```

Comment:
Could you also add examples with something like `seriesP @'[Hour, Minute]` where you have `1+1/2h` and `1+1/2m` and `1h1+1/2m`?